### PR TITLE
Add setup-devstack-swift option to test_and_publish_charm

### PR DESCRIPTION
**Pull request by amandahla**
Set setup-devstack-swift action to prepare a swift server for integration tests.  Default: false.

### CODE REVIEW

**Review by gtrkiller — DISMISSED** [review]
LGTM

File: .github/workflows/test_and_publish_charm.yaml [hunk 1]
```diff
@@ -28,6 +28,10 @@ on:
         description: List of testing modules to run the tests in JSON format, i.e. '["foo", "bar"]'. Each element will be passed to pytest through tox as -k argument
         type: string
         default: '[""]'
+      setup-devstack-swift:
```
Comment (arturo-seijas) [inline .github/workflows/test_and_publish_charm.yaml:31]: Can you update the README too?

Comment (amandahla) [reply .github/workflows/test_and_publish_charm.yaml:31]: Sure.

**Review by arturo-seijas — APPROVED** [review]
LGTM

**Review by gregory-schiano — APPROVED** [review]
LGTM, it's indeed required for your other PR

**Pull request merged**